packages/shared/src/components/post/FixedPostNavigation.tsx

```diff
@@ -22,15 +22,16 @@ function FixedPostNavigation({
       post={post}
       className={{
         container: classNames(
-          'fixed z-3 w-full bg-theme-bg-secondary border-b border-theme-divider-tertiary py-4 px-6 top-0 left-0 ml-0 laptop:left-[unset] laptop:-ml-8',
+          'fixed z-3 w-full bg-theme-bg-secondary border border-theme-divider-tertiary py-4 px-6 top-0 ml-0',
+          'laptop:left-[unset] max-w-full',
           className?.container,
         ),
         actions: className?.actions,
       }}
       onPreviousPost={onPreviousPost}
       onNextPost={onNextPost}
     >
-      <div className="overflow-hidden flex-col flex-1 ml-2">
+      <div className="hidden tablet:flex overflow-hidden flex-col flex-1 ml-2">
```

Comment:
The buttons should be right alligned
<img width="409" alt="image" src="https://user-images.githubusercontent.com/1488164/215416262-5ad5f44a-d45a-4309-870e-2ec79ac11416.png">

Reply:
Yep, didn't push the fix yet as I was working on something else. Should be committed now. Thank you!